src/Pages/Login/Components/InputComboBox.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+class InputComboBox extends Component {
+  render() {
+    const { name, userInfo, password, topText } = this.props;
+    return (
+      <InputTextBox>
+        <div className="InputComboBox">
+          <p className="inputTopText">
+            {topText}
+            <span>*</span>
+          </p>
+          <input
+            className="userInfoInputBox"
+            text={topText !== "Password" ? "text" : "password"}
+            value={topText !== "Password" ? userInfo : password}
+            name={name}
+            onChange={this.props.onChangeHandler}
+          />
```

Comment:
이렇게 해도 괜찮지만 가독성이 조금 떨어집니다. 아래와 같이 하시면 삼항연산자를 한 번만 쓸 수 있습니다!
```suggestion
      {topText === "Password" ? 
      (
      <input
            className="userInfoInputBox"
            text={"password"}
            value={password}
            name={name}
            onChange={this.props.onChangeHandler}
          />
          ) : (
                <input
            className="userInfoInputBox"
            text={"text"}
            value={userInfo}
            name={name}
            onChange={this.props.onChangeHandler}
          />)
```

Reply:
궁금한건 이부분이 가독성이 무슨 로직이 되는지는 이해가 좋은데, 문장이 너무 길어지는것같아서 의문이 생깁니다. 길어지는것과 무관한 가독성은 무관한건가요?

Reply:
어느 쪽에 우선순위를 두느냐에 따라 다른데 저 같은 경우는 짧게 줄인다고 해서 다 좋다고 생각하지는 않는 편입니다.
저는 확장성을 생각했을 때 두 input의 props에 삼항 연사자가 더 들어가면
차라리 한 번의 삼항연산자로 뺀 다음에 input을 더 자유롭게 사용할 수 있다고 생각했습니다.
그래서 이 부분은 정답은 없는 것 같습니다! 규영님이 규영님의 코드 스타일을 갖춰 나가시는 게 더 좋은 것 같습니다!